sunpy/visualization/imageanimator.py

```diff
@@ -954,11 +957,25 @@ def __init__(self, data, wcs=None, image_axes=[-1, -2], unit_x_axis=None, unit_y
         list_slices_wcsaxes[image_axes[0]] = 'x'
         list_slices_wcsaxes[image_axes[1]] = 'y'
         self.slices_wcsaxes = list_slices_wcsaxes[::-1]
+
+        if(unit_x_axis is not None):
+            self.unit_x_axis = u.Unit(unit_x_axis)
```

Comment:
If the user input is not None, then its get converted to the astropy units.
Like for any naive input for example `meters` , `self.unit_x_axis = u.m` .